### PR TITLE
MAR-2450. Added the anchor link to Mad Community

### PR DESCRIPTION
**Pull request by TangoPJ**
the task: https://maddevs.atlassian.net/browse/MAR-2450
1. Added the anchor link to the component
2. Added the logic to copy internal link
3. Updated snapshots

image

### CODE REVIEW

File: client/assets/styles/_global.scss [hunk 1]
```diff
@@ -447,3 +446,78 @@
...
+  }
+
+  &:hover {
+    .anchor_copy-link button {
+      display: block;
+    }
+  }
+}
+
+.customer-university-title {
```
Comment (kotyunin) [inline client/assets/styles/_global.scss:505]: А зачем нам .customer-university-title в глобальных стилях? Мы же его не переиспользуем

Comment (TangoPJ) [reply client/assets/styles/_global.scss]: Done

**Review by kotyunin — APPROVED** [review]

File: client/components/Blog/shared/СustomerUniversityTitle.vue [hunk 2]
```diff
@@ -20,27 +55,27 @@ export default {
...
+    },
+  },
 
-  @media screen and (max-width: 768px) {
-    @include font('Poppins', 50px, 700);
-    line-height: 104%;
-    margin: 42px 0 16px 0;
-  }
+  methods: {
+    copyAnchorLink(event) {
```
[the remark this answers is not in the document]
Comment (TangoPJ) [reply client/components/Blog/shared/СustomerUniversityTitle.vue]: Done

**Review by RomanChasovitin — APPROVED** [review]
approved

**Pull request merged**